doc/source/about.md

```diff
@@ -0,0 +1,38 @@
+# About GeoPandas
+
+```{toctree}
+:maxdepth: 2
+:caption: About
+:hidden:
+
+Roadmap <about/roadmap>
+Team <about/team>
+Citing <about/citing>
+Logo <about/logo>
+```
+
+
+GeoPandas is a project to add support for geographic data to pandas objects.
+It currently implements `GeoSeries` and `GeoDataFrame` types which are subclasses
+of `pandas.Series` and `pandas.DataFrame` respectively. GeoPandas objects can act
+on `shapely` geometry objects and perform geometric operations.
+
```

Comment:
Here, we should probably also say something like "GeoPandas is an open source project ..." ?

Reply:
In addition, we could maybe put also here a short version of what is in the community page: "GeoPandas is a community-led project written, used and supported by a wide range of people from all around of world of a large variety of backgrounds." and then maybe with something like "Want to get involved in the community? See [link to community section]"

Reply:
For my first comment here, numpy "about" page has the sentence "NumPy will always be 100% open source software, free for all to use and released under the liberal terms of the modified BSD license."